fill/sensors/load_cell.cc

```diff
@@ -1,5 +1,14 @@
 #include "load_cell.h"
 
-uint16_t LoadCell::Read(Adafruit_ADS1015* adc) {
-    return adc->readADC_SingleEnded(LC_CHANNEL);
+LoadCell::LoadCell(int address, uint8_t adc_channel){
+    adc = new Adafruit_ADS1015(address);
```

Comment:
We should probably refactor this and the PTs to take an Adafruit_ADS1015 object pointer in the constructors so that we don't have duplicate objects for this

Reply:
Then we can create, begin, and maybe set gain just once outside of these classes